packages/tailwind/src/kz-spacing.ts

```diff
@@ -1,28 +1,14 @@
-import { defaultTheme } from "@kaizen/design-tokens"
+const { heartTheme } = require("@kaizen/design-tokens")
```

Comment:
```suggestion
import { heartTheme } from "@kaizen/design-tokens"
```

Using `require` is an old pattern. Did it not work with `import`?

Reply:
It was throwing an error... But not any more. Weird.

Changed to import now